src/adapters/Auth/index.test.ts

```diff
@@ -0,0 +1,62 @@
+import requestManager from 'lib/requestManager';
+
+import AuthAdapter from './';
+
+jest.mock('lib/requestManager', () => jest.fn());
+
+describe('AuthAdapter', () => {
+  describe('login()', () => {
+    it('fires post request to login api with the correct data', () => {
```

Comment:
```suggestion
    it('fires a post request to the login API with the correct data', () => {
```

Please check other examples as well